target_chains/ethereum/contracts/contracts/pyth/PythState.sol

```diff
@@ -38,6 +38,9 @@ contract PythStorage {
         // Mapping of cached price information
         // priceId => PriceInfo
         mapping(bytes32 => PythInternalStructs.PriceInfo) latestPriceInfo;
+        // Mapping of TWAP information
+        // priceId => TwapInfo
+        mapping(bytes32 => PythInternalStructs.TwapInfo) latestTwapInfo;
```

Comment:
I don't think it makes sense to store these updates given that there isn't a notion of "latest". Instead, let's implement this whole PR along the lines of parsePriceFeedUpdates where the data is parsed, validated, and returned to the caller without being stored by the contract.